pyproject.toml

```diff
@@ -46,6 +46,8 @@ seaborn = "*"
 pyarrow = "*"
 implicit = ">=0.5"
 pytorch-ranger = "^0.1.1"
+nmslib = "*"
```

Comment:
to discuss: shall we add ann inference as an `extra`?

Reply:
> to discuss: shall we add ann inference as an `extra`?

do you mean dependencies of ANN
```
[tool.poetry.extras]
ann = ["nmslib", "hnswlib"]
```
or ANN as an extra package?
```
[tool.poetry.extras]
ann = ["replay_ann"]
```
?

I think in both cases, no (in current implementation), because ANN functionality is directly mixed in with the "main" models.